src/discord/Message.ts

```diff
@@ -12,5 +12,5 @@ Message.prototype.hasPrefix = function hasPrefix(prefix) {
 };
 
 Message.prototype.isDirectMessage = function isDirectMessage() {
-  return this.channel instanceof DMChannel || this.channel instanceof GroupDMChannel;
```

Comment:
Pretty sure there is a replacement for `GroupDMChannel´ which we need here.

Seems like there is [something else](https://discord.js.org/#/docs/main/master/class/Channel?scrollTo=type) we can use.